.github/workflows/release-edc.yml

```
@@ -0,0 +1,76 @@
+name: Create EDC Release
+on:
+  workflow_dispatch:
+    inputs:
+      edc_version:
+        description: 'Version string that is used for publishing. Appending -SNAPSHOT will create a snapshot release.'
```

Maybe add an example to remind people of the correct formatting?

```suggestion
        description: 'Version string that is used for publishing (e.g. "1.0.0", NOT "v1.0.0"). Appending -SNAPSHOT will create a snapshot release.'
```